src/gui/plugins/component_inspector/ComponentInspector.cc

```diff
@@ -728,6 +747,24 @@ void ComponentInspector::OnPose(double _x, double _y, double _z, double _roll,
   this->dataPtr->node.Request(poseCmdService, req, cb);
 }
 
+/////////////////////////////////////////////////
+void ComponentInspector::OnPhysics(double _stepSize, double _realTimeFactor)
+{
+  std::function<void(const ignition::msgs::Boolean &, const bool)> cb =
+      [](const ignition::msgs::Boolean &/*_rep*/, const bool _result)
+  {
+    if (!_result)
+        ignerr << "Error setting physics parameters" << std::endl;
+  };
+
+  ignition::msgs::Physics req;
+  req.set_max_step_size(_stepSize);
+  req.set_real_time_factor(_realTimeFactor);
+  auto physicsCmdService = "/world/" + this->dataPtr->worldName
```

Comment:
check the topic -> transport::TopicUtils::AsValidTopic()

Reply:
Done!